packages/engine/Source/Renderer/AutomaticUniforms.js

```diff
@@ -939,6 +939,59 @@ const AutomaticUniforms = {
     },
   }),
 
+  /**
+   * An automatic GLSL uniform containing the ellipsoid surface normal
+   * at the position below the eye (camera), in eye coordinates.
+   * This uniform is only valid when the {@link SceneMode} is <code>SCENE3D</code>.
```

Comment:
I'm guessing this is why in CV mode terrain exaggeration isn't working?

Note that vertical exaggeration for the globe does work in 2D/CV mode.

Reply:
@ptrgags I looked at CV mode a little. I'm relatively unfamiliar with how we implement this. But supporting CV and 2D modes does look like a significant lift, perhaps out of scope for this PR. I opened https://github.com/CesiumGS/cesium/issues/11721 to continue the discussion there.